nextpipe/decorators.py

```diff
@@ -798,7 +798,7 @@ def __init__(
         app_id : str
             The ID of the Nextmv Application to run.
         instance_id : str, optional
-            The ID of the instance to run, by default "devint".
+            The ID of the instance to run, uses default instance by default.
```

Comment:
Nit: "uses default instance by default" is clear in context but syntactically odd (to me at least). I would resolve this by being more verbose, but this might not be the best way.

```suggestion
            The ID of the instance to run, uses the running app's default instance by default.
```

Reply:
Possibly align with `app` def below?

```suggestion
            The ID of the instance to run. Default is defined by the app on Platform.
```